tp_timesheet/__main__.py

```diff
@@ -76,13 +70,25 @@ def parse_args():
     parser.add_argument(
         "-t",
         "--task",
-        type=str,
-        default="live",
-        choices=["holiday", "live", "OOO", "training"],
+        action="append",
+        nargs=2,
+        metavar=("task", "hour"),
         help="The type of task for the clockify submission. Specify task name "
-        + "if it is anything other than 'live'. Put the task name (e.g., training, OOO, holiday)",
+        + "if it is anything other than 'live'. Put the task name(idle, training, issue, OOO, holiday).\n"
+        + 'Passing multiple task-hour pair is also acceptable. (ex, "--task live 4 --task OOO 4" for afternoon OOO.)',
     )
-    return parser.parse_args()
+    args = parser.parse_args()
+
+    # postprocessing args
+    args.task = [["live", "8"]] if not args.task else args.task  # default value
+    args.task = {k: int(v) for (k, v) in args.task}
```

Comment:
```suggestion
    args.task = {task_name: int(hours) for (task_name, hours) in args.task}
```
Just to be more obvious about what the tuple contents are.